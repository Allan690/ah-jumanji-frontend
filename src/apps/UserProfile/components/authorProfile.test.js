import React from 'react';

import { shallow } from 'enzyme';
import AuthorProfile from './authorProfile';




describe('authorProfile page', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<AuthorProfile debug />);

    expect(component).toMatchSnapshot();
  });

});
