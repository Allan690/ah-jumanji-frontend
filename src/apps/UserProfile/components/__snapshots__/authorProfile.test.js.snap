// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`authorProfile page should render correctly in "debug" mode 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
